src/scandeval/sequence_classification.py

```diff
@@ -298,7 +298,7 @@ def _extract_labels_from_generation(
 
 
 def get_closest_logprobs_labels(
-    generation_logprobs: tuple[torch.Tensor],
+    generation_logprobs: tuple[torch.Tensor] | torch.Tensor,
```

Comment:
This seems like something that should have been standardized before this function.